pkg/common/args.go

```diff
@@ -88,13 +118,6 @@ func ValidateConformanceArgs() error {
 		}
 	}
 
-	log.Printf("Test framework will start %d thread(s) and use verbosity level %d.",
-		viper.Get("parallel"), viper.Get("verbosity"))
-
-	outputDir := viper.GetString("output-dir")
```

Comment:
removing L94-97 does not seem right, do we not need this?

Reply:
I didn't remove those lines, I moved them from this function into `SetDefaults`, where all the other status printing happens. Seems more natural to have all the validation in one func and all the information printing in the other.